### ISSUE TITLE
FMC, SUPPLY AND DEMAND ISSUES

### ISSUE THREAD

**Issue by CaiganMythFang** [CaiganMythFang/TheFerretBusinessIssues#8]
Here are the current known issues and their status for : FMC Values, Crating Goods, Shipping Goods, Payment from Shipping Goods.
## FMC Value Issues

The following either need to be revalued or contain a bug that needs to be fixed for next update
- [x] **Bugged**
  - [x] Bibliocraft
    - [x] Framing Saw
      - Removed from FMC Values
- [x] **Needs Revaluing**
  - [x] Thaumcraft
  - [x] MineFactory Reloaded
## Supply and Demand issues.
- [ ] Crating Goods
  - [ ] Blank
- [x] Supply Crate Combinations
  - [x] 200 + 300 = 500
  - [x] 200 + 300 + 500 = 1000
- [x] Shipping Crates
  - [x] 200,000 Crate not shippable
    - [x] Add shipping quest
    - [x] Add SAGMill recipe
  - [x] 200,000 Crate cannot be broken down to smaller crates

**Comment by KahnWolfe** [comment]
I had some questions based on some of the things I have happened across with parts being worth more than their sum total. Most of which I am looking at were fairly vanilla things. Glowstone for example. Each glowstone is worth 300, making four worth 1200 while a glowstone block is only worth 1000. Similar with things like redstone lamps. Only 1000 in value while the total of the parts is worth 1200 (or more if you break down the glowstone in an appropriate manner)

**Comment by CaiganMythFang** [comment]
Currently, FMC is rounded to the nearest proper value. The values are
.25
.5
1
2
5
10
25
50
100
Every +100 to 500
Every +500 to 100,000
Every +1,000

So a crafted item that is 1,200 gets rounded down to 1,000. A crafted item that is 1,250 gets rounded up to 1,500. Overall, it averages out since some items become worth more, while others become worth less.

This is so there isn't 10x more recipes within the minetweaker scripts(a machine might use cobblestone as a component, for example). It isn't worth the extra complexity, in essence.

**Comment by KahnWolfe** [comment]
Fair enough. Was just curious as to how that was working and if there was any consideration given to evaluations. That does greatly make sense though and must save a great deal of coding. Will take that information and probably start working on selling lower value items rather than losing out by making more complex things. Tis a bit sad that it seems to kind of punish effort for crafting, but it's all part of playing the system!
